sdk1/src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/encryption/DynamoDBEncryptor.java

```diff
@@ -291,6 +301,11 @@ public static DynamoDBEncryptor getInstance(EncryptionMaterialsProvider provider
     return itemAttributes;
   }
 
+  private boolean isAnyKeyMarkedWithEncryptionFlags(
+      Set<String> keysToCheck, Map<String, Set<EncryptionFlags>> attributeFlags) {
+    return keysToCheck.stream().anyMatch(key -> !attributeFlags.get(key).isEmpty());
```

Comment:
naming nitpick: `keys` should be referred to as `attributeNames` for consistency since `itemAttributes` is modelling DDB items.

Reply:
done